schemas/data/propensity.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/propensity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Propensity",
+  "type": "object",
+  "description":
+    "The propensity schema represents the likelihood of a certain action happening in a certain window of time. The propensity schema can host various predictive results e.g churn. This is a general-purpose schema that is designed to contain the results of any propensity scoring algorithm.",
+  "definitions": {
+    "propensity": {
+      "properties": {
+        "xdm:score": {
```

Comment:
Elsewhere we are using `xdm:confidence` (in `EndUserIDs`) and we should be using it in Segment Assignment. I'm fine using the name "Score" in the title or description, but we should keep all examples of modeling confidence intervals the same.